tests/PHPUnit/Integration/Tracker/UserIdVisitorIdTest.php

```diff
@@ -780,6 +780,10 @@ private function assertVisitorIdsCount($expected)
     {
         $visitorIdsCount = Db::fetchOne("SELECT COUNT(DISTINCT idvisitor) FROM " . Common::prefixTable('log_visit'));
         $this->assertEquals($expected, $visitorIdsCount);
+
+        // check that idvisitor column was updated.
+        $visitorIdsCount = Db::fetchOne("SELECT COUNT(DISTINCT idvisitor) FROM " . Common::prefixTable('log_link_visit_action'));
+        $this->assertEquals($expected, $visitorIdsCount);
```

Comment:
Should we perhaps iterate here over all the tables returned by the provider and check the numbers in all of them? It would make sure that if we add or remove the column in future to/from a table it wouldn't go unnoticed that the update does/doesn't work there.